src/amo/components/AddonsByAuthorsCard/styles.scss

```diff
@@ -20,7 +20,7 @@
       display: grid;
       grid-auto-flow: initial;
       grid-gap: 6px;
-      grid-template-columns: repeat(3, 1fr);
+      grid-template-columns: repeat(3, minmax(32%, 1fr));
```

Comment:
I'm not sure what adding the `minmax` for this `horizontal` class does for us? What is the specific case we are protecting against?

Reply:
hey @bobsilverberg - i was seeing some weirdness at smaller breakpoints:

![Alt text](https://monosnap.com/image/mOt9NsgMrXWy9ZDLzzSLeeAUJO9ovc.png)

this seems to do similar thing as original style except it gives it a min so cols are distributed evenly even in ~ mobile where this didn't seem to be happening

Reply:
Thanks for the example, @rebmullin. That does illustrate what the problem is, but I still don't understand why adding a min column size fixes that. Do you know why?

Reply:
honestly i am not too sure why the initial code doesn't work but by using this minmax the min val enforces cols size (specifically col 2 and 3) to be redistributed as expected. There are probably different ways to approach this - with max value too or via fit-content.